lesson_25/james_capparell/src/components/header/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import './Header.scss';
+import logoImg from '@/assets/logo.png';
+import React from 'react';
+
+export const Header: React.FC = () => {
+  return (
+    <header className="header">
+      <div className="header-logo">
+        <a href="/">
+          <img src={logoImg} alt="Code Differently Logo" />
+        </a>
+      </div>
+      <ul className="header-top-menu">
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Contact</a>
+        </li>
+        <li>
+          <a href="@/pages/Home/AddProgram.tsx">Add Program</a>
```

Comment:
Link not working. You should probably update the router in `main.tsx` and update the path here.